src/status_im/translations/ja.cljs

```diff
@@ -158,83 +177,204 @@
    :enter-address                         "アドレスを入力してください"
    :more                                  "もっと"
 
-   ;group-settings
+   ;;group-settings
    :remove                                "削除"
    :save                                  "保存"
    :delete                                "削除"
    :clear-history                         "履歴の消去"
    :mute-notifications                    "お知らせをミュート"
    :leave-chat                            "チャットから退出"
    :chat-settings                         "チャット設定"
-   :edit                                  "偏執"
+   :edit                                  "編集"
    :add-members                           "メンバーを追加"
 
-   ;commands
+   ;;commands
    :chat-send-eth                         "{{amount}} ETH"
 
-   ;new-group
+   ;;location command
+   :your-current-location                 "あなたの現在の場所"
+   :places-nearby                         "近くの場所"
+   :search-results                        "検索結果"
+   :dropped-pin                           "ドロップされたピン"
+   :location                              "場所"
+   :open-map                              "地図を開く"
+   :sharing-copy-to-clipboard-address     "住所をコピーする"
+   :sharing-copy-to-clipboard-coordinates "座標をコピーする"
+
+   ;;new-group
    :new-group                             "新しいグループ"
    :reorder-groups                        "グループを並び替え"
-   :edit-group                            "グループを偏執"
+   :edit-group                            "グループを編集"
    :delete-group                          "グループを削除"
    :delete-group-confirmation             "このグループを削除します。連絡先には影響を与えません。"
    :delete-group-prompt                   "連絡先には影響を与えません。"
    :contact-s                             {:one   "連絡先"
                                            :other "連絡先"}
-   ;participants
 
-   ;protocol
+   ;;protocol
    :received-invitation                   "チャット招待状を受信しました"
    :removed-from-chat                     "グループチャットから削除されました"
    :left                                  "残り"
```

Comment:
@andytudhope The word "left" here means someone left the chat, right?

Reply:
Yeah, that is correct.